Purchases/Misc/Deprecations.swift

```diff
@@ -0,0 +1,119 @@
+//
+//  Copyright RevenueCat Inc. All Rights Reserved.
+//
+//  Licensed under the MIT License (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      https://opensource.org/licenses/MIT
+//
+//  Deprecations.swift
+//
+//  Created by Nacho Soto on 11/15/21.
+
+import StoreKit
+
+// swiftlint:disable unused_setter_value
+
+extension CustomerInfo {
+
+    /// Returns all product IDs of the non-subscription purchases a user has made.
+    @available(*, deprecated, message: "use nonSubscriptionTransactions")
```

Comment:
same thought for this method - it's currently just deprecated, not obsoleted, so it's still usable by customers

Reply:
Oops, you're absolutely right, I didn't look closely enough to notice that not all of these are obsoleted and some are deprecated. I'll rename to Obsoletions and make sure the `deprecated` ones are left untouched 👍